openapi-core/src/main/java/io/ballerina/openapi/core/generators/service/BallerinaServiceGenerator.java

```diff
@@ -247,14 +261,59 @@ private List<Node> applyFiltersForOperations(Filter filter, String path,
                 List<Node> relativeResourcePath = GeneratorUtils.getRelativeResourcePath(path, operation.getValue(),
                         resourceFunctionDocs);
                 // function call
-                FunctionDefinitionNode resourceFunction = getResourceFunction(operation, relativeResourcePath,
-                        path, resourceFunctionDocs);
+                FunctionDefinitionNode resourceFunction = generateWithoutDataBinding ?
+                        generateGenericResourceFunctions(operation,
+                                relativeResourcePath, path, resourceFunctionDocs) : getResourceFunction(operation,
+                        relativeResourcePath, path, resourceFunctionDocs);
```

Comment:
Do we support low level generation for remote functions as well?

Reply:
In service generation only resource functions are generated